primitives/src/runtime_api.rs

```diff
@@ -220,6 +225,6 @@ sp_api::decl_runtime_apis! {
 
 		/// Returns all onchain disputes.
 		#[api_version(3)]
```

Comment:
nit: The base version of the API is 3, so this is no longer needed.
```suggestion
```

Reply:
Are you sure it should be removed right here, in `decl_runtime_apis!()`? It seemed to me that it should be removed in implementations but kept in the declaration.

Reply:
Ah, I see, it's the attribute for `disputes()`, not the top-level one. Didn't understand it from code suggestions ;)